app/page.tsx

```diff
@@ -1,16 +1,38 @@
-import { GetTranscriptButton } from "./components/getTranscriptButton"
+"use client"
+import { useEffect, useState } from "react"
+import Cookies from "js-cookie"
+import { useRouter } from "next/navigation"
 
-const { TranscriptCard } = require("./components/TranscriptCard")
-const { SelectedTranscript } = require("./components/SelectedTranscript")
+import { Button } from "./components/Button"
+import { TranscriptCard } from "./components/TranscriptCard"
+import { SelectedTranscript } from "./components/SelectedTranscript"
 
 export default function Home() {
+  const router = useRouter()
+  const [token, setToken] = useState("")
```

Comment:
how come you're putting the token in state here? as far as i can see you're not using it anywhere else

Reply:
Oh good spot, this was leftover from some previous cookie handling but is no longer needed